cmd/osm-bootstrap/crds/config_multicluster_service.tpl

```diff
@@ -0,0 +1,167 @@
+# Custom Resource Definition (CRD) for OSM's multicluster config specification.
```

Comment:
These templates can all be removed, right?

Reply:
Ah, yes they can. Undoing a rename in git isn't as intuitive as I thought :)